src/Codeception/Module/WebDriver.php

```diff
@@ -774,7 +774,7 @@ function ($c) {
             },
             $cookies
         );
-        $this->debugSection('Cookies', json_encode($this->webDriver->manage()->getCookies()));
+        $this->debugSection('Cookies', json_encode($this->getCookiesAsArray($this->webDriver->manage()->getCookies())));
```

Comment:
Suggestion: how about moving this into a `debugCookies` method which will do excatly the same: print cookies.
THis way you can reuse it accross different methods without code duplication